api/ocm/extensions/blobhandler/handlers/generic/maven/blobhandler.go

```diff
@@ -129,3 +140,53 @@ func (b *artifactHandler) StoreBlob(blob cpi.BlobAccess, resourceType string, hi
 	}
 	return access.New(url, coords.GroupId, coords.ArtifactId, coords.Version), nil
 }
+
+// PomGav defines gav information in a POM file.
+type PomGav struct {
+	GroupID    string `xml:"groupId"`
+	ArtifactId string `xml:"artifactId"`
```

Comment:
```suggestion
	GroupId    string `xml:"groupId"`
	ArtifactId string `xml:"artifactId"`
```

or is there a special golang naming convention?

Reply:
No, just conform to `ArtifactId` down below... :)

Reply:
WAIT. I thought I did rename it hahahahaha. So sorry.